render/gles2/texture.c

```diff
@@ -203,6 +202,7 @@ struct wlr_texture *wlr_gles2_texture_from_wl_drm(struct wlr_egl *egl,
 	texture->wl_drm = data;
 
 	EGLint fmt;
+	texture->wl_format = 0xFFFFFFFF; // texture can't be written anyways
```

Comment:
You can just leave this unset in these cases

Reply:
"Unset" means 0, which is a valid format (ARGB8888 iirc)

Reply:
Maybe it's worth it to add a `immutable` attribute

Reply:
That's implicitly stored via texture->type. `WLR_GLES2_TEXTURE_GLTEX` signals that it's mutable. And yep, that was my thought with not leaving it 0. We don't rely on this value anywhere btw i just didn't want to store something wrong.

Reply:
I just felt bad about setting an enum to a value outside the enum.